docs/DeveloperGuide.adoc

```diff
@@ -731,64 +731,70 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 
 |`* * *` |user |edit their contact details |I do not have to delete and create a new contact
 
+|`* * *` |user |edit my own contact details |I can make changes/correct my errors
+
 |`* * *` |user |add/edit a tag pegged to my contact |I can update the tags if necessary
 
 |`* * *` |user |undo and redo my last action |I can recover from my mistakes
 
-|`* * *` |user |know how to get to my contact's address from my current location on Google maps |I know which route to take to get there
-
-|`* * *` |user |be warned before deleting/ clearing contact |I won't accidentally delete a person
-
-|`* * *` |user |message/email/call my contacts in the list |I do not have to copy/paste/manually enter their details
-
 |`* * *` |user |create groups |I can sort my contacts according to social context
 
 |`* * *` |user |sort and find according to tags/social groups/name/dob |I can conveniently look out for specific tags/identify categories
```

Comment:
The two stories seem somewhat redundant. or at least one of them should have lower priority